libsolidity/analysis/ControlFlowBuilder.cpp

```diff
@@ -73,10 +74,43 @@ bool ControlFlowBuilder::visit(BinaryOperation const& _operation)
 			return false;
 		}
 		default:
-			return ASTConstVisitor::visit(_operation);
+		{
+			visitNode(_operation);
```

Comment:
This can now be pulled out of the switch.

Reply:
I will keep it here to avoid duplicated `visitNode` call due to return statement `return ASTConstVisitor::visit(_operation);`